src/tests/ftest/daos_test/daos_core_test.yaml

```diff
@@ -56,6 +56,7 @@ daos_tests:
     test_d:
       daos_test: d
       test_name: DAOS degraded-mode tests
+      scalable_endpoint: True
```

Comment:
why are we enabling sep for this specifically? verbs provider for example doesnt support SEP at all; before we can have verbs running in CI it would be better to test all daos tests with ofi+sockets using same params as we would expect customers to use eventually with verbs

Reply:
I may have misunderstood the [this](https://jira.hpdd.intel.com/browse/DAOS-4537?focusedCommentId=245070&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-245070) comment, but it states to "switch to SEP mode for this test".

Reply:
yeah, indeed (sheepish smile). I didn't realize this was for the original ticket, was thinking we are trying to resolve only the failure that started happening in last few days.